pubspec.yaml

```diff
@@ -10,21 +10,21 @@ environment:
 dependencies:
```

Comment:
I ran `flutter pub upgrade --major-versions` because `flutter pub get` failed and I could not start working on the project otherwise.